packages/@webex/plugin-meetings/test/unit/spec/locus-info/selfConstant.js

```diff
@@ -116,6 +116,7 @@ export const self = {
     audio: {
       muted: false,
       requestedToUnmute: false,
+      lastModifiedRequestedToUnmute: '2023-06-16T18:25:04.369Z',
```

Comment:
if the data we are receiving from the backend is ISO strings, i think we should consider converting them to a format that can be more easily compared.
```
Date.parse('2023-06-16T19:25:04.369Z')
```
outputs `1686943504369`